pytest_html/resources/main.js

```diff
@@ -12,18 +18,84 @@ function toArray(iter) {
 
 function find(selector, elem) {
     if (!elem) {
-        elem = document;
+        elem = DOCUMENT;
     }
     return elem.querySelector(selector);
 }
 
 function find_all(selector, elem) {
     if (!elem) {
-        elem = document;
+        elem = DOCUMENT;
     }
     return toArray(elem.querySelectorAll(selector));
 }
 
+function reset_sort_headers() {
+    //there isn't Unit Tests for this function
```

Comment:
Is there a reason we don't have tests for this? Perhaps there's a test coverage too we could run to see what's missing tests?